
From: Christoph Hellwig <hch@lst.de>

Only legit user is the partitioning code, in addition some uml code is
still using despite the uml people beeing told to fix it at least two
times.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/devfs/base.c |    1 -
 1 files changed, 1 deletion(-)

diff -puN fs/devfs/base.c~unexport-devfs_mk_symlink fs/devfs/base.c
--- 25/fs/devfs/base.c~unexport-devfs_mk_symlink	Tue Sep  7 16:29:25 2004
+++ 25-akpm/fs/devfs/base.c	Tue Sep  7 16:29:25 2004
@@ -1802,7 +1802,6 @@ static int __init devfs_setup(char *str)
 
 __setup("devfs=", devfs_setup);
 
-EXPORT_SYMBOL(devfs_mk_symlink);
 EXPORT_SYMBOL(devfs_mk_dir);
 EXPORT_SYMBOL(devfs_remove);
 
_
